ui-components/scripts/controllers/products.controller.js

```diff
@@ -52,7 +53,7 @@
       $scope.meta = cache.get('meta');
 
       if (angular.isUndefined($scope.products) && angular.isUndefined($scope.meta)) {
-        Api.products.query({
+        Api.products(token).query({
```

Comment:
passing the token - the same. I think all your http requests should have this. It could be done in one place for all requests